saleor/account/forms.py

```diff
@@ -127,3 +127,27 @@ def send_mail(
         # template, we remove it from the context.
         del context['user']
         emails.send_password_reset_email.delay(context, to_email)
+
+
+class EmailChangeForm(forms.Form):
+    """ Allow changing email """
+
+    email = forms.EmailField(
+        label=pgettext('New email', 'Email'), max_length=75)
```

Comment:
I think context could be a little more expanded here, like `Label of the user's email field`

Also, it's quite dangerous to limit the `max_length` at the form level, we need to make sure that this length is always the same as on the Model's field.